include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -2600,28 +2581,14 @@ namespace internal
       fe_degree > -1 ? static_dofs_per_face :
                        Utilities::pow(data.data.front().fe_degree + 1, dim - 1);
 
-    // we allocate small amounts of data on the stack to signal the compiler
-    // that this temporary data is only needed for the calculations but the
-    // final results can be discarded and need not be written back to
-    // memory. For large sizes or when the dofs per face is not a
-    // compile-time constant, however, we want to go to the heap in the
-    // `scratch_data` variable to not risk a stack overflow.
-    constexpr unsigned int stack_array_size_threshold = 100;
-
-    VectorizedArrayType *DEAL_II_RESTRICT temp1;
-    VectorizedArrayType
-      temp_data[static_dofs_per_face < stack_array_size_threshold ?
-                  2 * dofs_per_face :
-                  1];
-    if (static_dofs_per_face < stack_array_size_threshold)
-      temp1 = &temp_data[0];
-    else
-      temp1 = scratch_data;
+    VectorizedArrayType *temp1 = scratch_data;
 
     const unsigned int dummy = 0;
 
     // re-orientation
     std::array<const unsigned int *, n_face_orientations> orientation;
+    std::fill(orientation.begin(), orientation.end(), nullptr);
```

Comment:
Doesn't using `= {};` in the declaration in the line above have the same effect? I would find that slightly easier to read.